### PR TITLE
Detail reporting dashboard

### PR DESCRIPTION
**Pull request by MaximilianWagner**
Features:
* Implement detail reporting dashboard.
* Redirected from reporting dashboard.
* Enhance Table styling with bootstrap.

### CODE REVIEW

File: embark/templates/uploader/reportDashboard.html [hunk 1]
```diff
@@ -51,5 +53,13 @@
         $(document).ready(function() {
             $('#report-table').DataTable();
         } );
+
```
Comment (diegiesskanne) [inline embark/templates/uploader/reportDashboard.html:56]: can this be removed?

File: embark/uploader/views.py [hunk 2]
```diff
@@ -328,16 +341,21 @@ def get_load(request):
...
+    firmware_id = analyze_id
     if not firmware_id:
         logger.error('Bad request for get_individual_report')
         return JsonResponse(data={'error': 'Bad request'}, status=HTTPStatus.BAD_REQUEST)
     try:
         result = Result.objects.get(firmware_id=int(firmware_id))
-        result = model_to_dict(result)
-        result['strcpy_bin'] = json.loads(result['strcpy_bin'])
-        return JsonResponse(data=result, status=HTTPStatus.OK)
+        firmware = Firmware.objects.get(pk=int(firmware_id))
```
Comment (diegiesskanne) [inline embark/uploader/views.py:351]: maybe rename this, because otherwise the naming is confusing (firmware.firmware)

**Review by diegiesskanne — APPROVED** [review]
seems good

**Comment by MaximilianWagner** [comment]
Have a look here:
http://emba-env.ddns.net:8000/home/reportDashboard/
and here:
http://emba-env.ddns.net:8000/home/individualReportDashboard/1?

File: embark/uploader/views.py (hunk 2 above)
Comment (MaximilianWagner) [reply embark/uploader/views.py]: Done !

File: embark/templates/uploader/reportDashboard.html (hunk 1 above)
Comment (MaximilianWagner) [reply embark/templates/uploader/reportDashboard.html:64]: Its removed. Thanks.

**Pull request merged**